crates/oracle/src/error_handling.rs

```diff
@@ -8,46 +8,3 @@ pub type OracleControlFlow = ControlFlow<(), Option<Duration>>;
 pub trait MainLoopFlow {
     fn instruction(&self) -> OracleControlFlow;
 }
-
-/// Helper function to convert a slice of items into a string, for use in [`Display`] contexts.
-pub fn format_slice(v: &[impl Display]) -> String {
-    let mut s = "[".to_string();
-    s.push_str(
-        v.iter()
-            .map(ToString::to_string)
-            .collect::<Vec<_>>()
-            .join(", ")
-            .as_str(),
-    );
-    s.push(']');
-    s
-}
```

Comment:
We don't currently use this.

If we were to use it again, a simpler third-party crate like [`slicedisplay`](https://github.com/vrmiguel/slicedisplay) would suffice.